travis/getaddons.py

```diff
@@ -71,7 +71,7 @@ def get_addons(path):
         res = [path]
     else:
         res = [os.path.join(path, x)
-               for x in os.listdir(path)
+               for x in sorted(os.listdir(path))
```

Comment:
Why do you want to make me spent 1 hour or more to produce a print screen on something so obvious? Is this really necessary for you evaluation of this changeset (please do not burden upon me unrelated research questions).